### ISSUE TITLE
Need latest hadolint v1.10.4 in docker image

### ISSUE THREAD

**Issue by max-10001** [hadolint/hadolint#249]
- [] This is a bug report
- [ x] This is a feature request
- [x ] I searched existing issues before opening this one

### Expected behavior
v1.10.4 of hadolint in docker image
### Actual behavior
Haskell Dockerfile Linter v1.10.3-1-g69be604
### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.10.3-1-g69be604
```

**Dockerfile** (if relevant)

```Dockerfile
(paste content of your Dockerfile)
```

**Additional environment details (OS, stack version, etc.)**

**Comment by lorenzo** [comment]
Please run `docker pull hadolint/hadolint:latest` so you get the newest version. I'm going to close the ticket since the latest tag is pointing currently to `v1.10.4`
